--- /dev/null/OnboardGtk.py	2020-02-23 01:46:34.527590513 -0800
+++ /usr/lib/python3.8/site-packages/Onboard/OnboardGtk.py	2020-02-23 01:39:15.890940319 -0800
@@ -272,7 +272,6 @@
         self.keyboard_widget.set_startup_visibility()

         # keep keyboard window and icon palette on top of dash
-        self._keep_windows_on_top()

         # connect notifications for keyboard map and group changes
         self.keymap = Gdk.Keymap.get_default()
@@ -646,19 +645,13 @@
         self.reload_layout()

     def _keep_windows_on_top(self, enable=True):
-        if not config.xid_mode:  # be defensive, not necessary when embedding
-            if enable:
-                windows = [self._window, self._window.icp]
-            else:
-                windows = []
-            _logger.debug("keep_windows_on_top {}".format(windows))
-            self._osk_util.keep_windows_on_top(windows)
+        pass

     def on_focusable_gui_opening(self):
-        self._keep_windows_on_top(False)
+        pass

     def on_focusable_gui_closed(self):
-        self._keep_windows_on_top(True)
+        pass

     def reload_layout_and_present(self):
         """
